### DISCUSSION TITLE
Countdown Timer with bar/circle

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#8979] [Help] [answered]
Hi,

since you can't add <script>, how would I add a countdown timer with a bar or circle?
There are some nice components with the timer but I need the bar for my project.
Any ideas?

**Comment by melohagan** [comment, marked as answer]
Hey @kasgithub8 

You could check out this custom component: https://github.com/JayP718/countdown_timer_bb_plugin

It doesn't support an animated bar/circle but you could always raise an issue there, or try forking the project if you are comfortable with a bit of programming. 

Jay is quite an active member of the community, and you'll probably be able to find him on the discord channel as well! Of course there's no obligation for him to help.

**Reply by kasgithub8** [reply]
Hi @melohagan,

thanks! Hopefully Jay answers me. 😄
